### ISSUE TITLE
Node formula gives npm v6.13.2, which is vulnerable

### ISSUE THREAD

**Issue by MikeTaylor** [Homebrew/homebrew-core#47969]
The `node` package's formula at https://github.com/Homebrew/homebrew-core/blob/master/Formula/node.rb specifies https://registry.npmjs.org/npm/-/npm-6.13.2.tgz

See the security advisory at https://blog.npmjs.org/post/189618601100/binary-planting-with-the-npm-cli which says "tl;dr - Update to npm v6.13.4 as soon as possible on all your systems to fix a vulnerability allowing arbitrary path access."

My apologies if this is not the right place to report the advisory — I can't find any indication of a dedicated channel for such reports. I have made some attempt to fill out the issue-filing boilerplate below.

- [**YES** ] ran `brew update` and can still reproduce the problem?
- [**YES**] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [**YES** ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link? **(No output link: it just says "Error: No logs.")**
- [**NO** ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue? **(This is obviously not relevant in the present case)**

## What you were trying to do (and why)

Upgrade to a secure version of `npm`.

## What happened (include command output)

```
ringo:Downloads$ brew upgrade npm
Warning: npm 13.3.0 already installed
ringo:Downloads$ npm -version
6.13.2
ringo:Downloads$ 
```

**Comment by bayandin** [comment]
Fixed in #47940

**Comment by MikeTaylor** [comment]
Excellent, thank you!
